plugin/cypress/e2e/featureFiles/navigation.feature

```diff
@@ -8,16 +8,21 @@ Feature: Kiali integration with OCP Console
   Background:
     Given user is logged as administrator in OCP Console
     And user clicks on the Service Mesh icon in the left navigation bar
+    And cypress intercept hooks are registered
 
   Scenario: Service mesh buttons are displayed
     Then buttons for Overview, Graph and Istio Config are displayed
 
   Scenario: Overview page is displayed correctly
-    Then user is redirected to the OSSMC "Overview" page
+    When user is redirected to the OSSMC "Overview" page
+    And user sees istio-system overview card
+    Then user sees memory and cpu charts from Kiali
```

Comment:
```suggestion
    Then user sees istio-system overview card
    And user sees memory and cpu charts from Kiali
```
What about something like this? I think that user sees istio-system overview card is part of Then, and it is executed before user sees memory and cpu charts test step.

Reply:
good suggestion, committed